modules/manage/partials/authentication.adoc

```diff
@@ -687,6 +690,46 @@ pandaproxy_client:
 ----
 endif::[]
 
+[[plain]]
+==== SASL/PLAIN
+
+You can configure Kafka clients to authenticate using either SASL/SCRAM or SASL/PLAIN with a
+single account using the same username and password. Unlike SASL/SCRAM, which uses a challenge response with hashed
+credentials, SASL/PLAIN transmits plaintext passwords. While not required, it is recommended that you use TLS for external encryption when using SASL/PLAIN authentication.
+
+If you have existing PLAIN Kafka clients and applications, you can migrate to Redpanda without updating
+your application by creating local Redpanda SCRAM accounts and enabling PLAIN as an authentication mechanism.
+
+NOTE: Clusters configured with _only_ a SASL/PLAIN mechanism are not recommended for production.
```

Comment:
It's not possible:
```
rpk cluster config set sasl_mechanisms '["OAUTHBEARER","PLAIN"]' 
No changes were made: Validation errors:
 * sasl_mechanisms: SCRAM mechanism must be enabled if PLAIN is enabled
 ```

Reply:
So switch back to just saying it is not supported @BenPope ?